File: C++11/백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.cc
#include <stdio.h>

int main(){
    int a,b,c;
    scanf("%d %d %d",&a,&b,&c);
    if(a+b+c<100){
        int min = a<b?a:b;
        min = min<c?min:c;
        if(min==a){
            printf("Soongsil");
        }
        else if(min==b){
            printf("Korea");
        }
        else{
            printf("Hanyang");
        }
    }
    else{
        printf("OK");
    }
}

File: Java/백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] a = new int[3];
        for (int i=0; i<3; i++){
            a[i] = in.nextInt();
        }
        if(a[0]+a[1]+a[2]<100){
            int min = a[0]<a[1]?a[0]:a[1];
            min = min<a[2]?min:a[2];
            if(min==a[0]){
                System.out.println("Soongsil");
            }
            else if(min==a[1]){
                System.out.println("Korea");
            }
            else{
                System.out.println("Hanyang");
            }
        }
        else{
            System.out.println("OK");
        }
    }
}